src/OpenTelemetry.Instrumentation.Wcf/README.md

```diff
@@ -214,26 +210,7 @@ contracts you want to instrument:
 
 ## Known issues
 
-WCF library does not provide any extension points to handle exception thrown on
-communication (e.g. EndpointNotFoundException). Because of that in case of such
-an event the `Activity` will not be stopped correctly. This can be handled in
-an application code by catching the exception and stopping the `Activity` manually.
-
-```csharp
-    StatusResponse? response = null;
-    try
-    {
-        response = await client.PingAsync(statusRequest).ConfigureAwait(false);
-    }
-    catch (Exception)
-    {
-        var activity = Activity.Current;
-        if (activity != null && activity.Source.Name.Contains("OpenTelemetry.Instrumentation.Wcf"))
-        {
-            activity.Stop();
-        }
-    }
-```
+None
```

Comment:
Remove whole Known issues section.

There is no common practice to keep it empty.